gui/default/index.html

```diff
@@ -391,6 +391,48 @@ <h4 class="panel-title">
                 <div class="panel-body">
                   <table class="table table-condensed table-striped table-auto">
                     <tbody>
+                      <tr class="visible-xs">
+                        <th ng-switch="folderStatus(folder)">
+                          <span ng-switch-when="paused" class="fas fa-fw fa-pause"></span><span
+                          ng-switch-when="unknown" class="fas fa-fw fa-question-circle"></span><span
+                          ng-switch-when="unshared" class="fas fa-fw fa-unlink"></span><span
+                          ng-switch-when="scan-waiting" class="fas fa-fw fa-hourglass-half"></span><span
+                          ng-switch-when="cleaning" class="fas fa-fw fa-recycle"></span><span
+                          ng-switch-when="clean-waiting" class="fas fa-fw fa-hourglass-half"></span><span
+                          ng-switch-when="stopped" class="fas fa-fw fa-stop"></span><span
+                          ng-switch-when="scanning" class="fas fa-fw fa-search"></span><span
+                          ng-switch-when="idle" class="fas fa-fw fa-check"></span><span
+                          ng-switch-when="localadditions" class="fas fa-fw fa-check"></span><span
+                          ng-switch-when="sync-waiting" class="fas fa-fw fa-hourglass-half"></span><span
+                          ng-switch-when="sync-preparing" class="fas fa-fw fa-hourglass-half"></span><span
+                          ng-switch-when="syncing" class="fas fa-fw fa-sync"></span><span
+                          ng-switch-when="outofsync" class="fas fa-fw fa-exclamation-circle"></span><span
+                          ng-switch-when="faileditems" class="fas fa-fw fa-exclamation-circle"></span><span
+                          ng-switch-when="localunencrypted" class="fas fa-fw fa-exclamation-circle"></span>&nbsp;<span
+                          translate>Folder Status</span>
+                        </th>
+                        <td ng-switch="folderStatus(folder)" class="text-right">
+                          <span ng-switch-when="paused" translate>Paused</span>
+                          <span ng-switch-when="unknown" translate>Unknown</span>
+                          <span ng-switch-when="unshared" translate>Unshared</span>
+                          <span ng-switch-when="scan-waiting" translate>Waiting to Scan</span>
+                          <span ng-switch-when="cleaning" translate>Cleaning Versions</span>
+                          <span ng-switch-when="clean-waiting" translate>Waiting to Clean</span>
+                          <span ng-switch-when="stopped" translate>Stopped</span>
+                          <span ng-switch-when="scanning" translate>
```

Comment:
This repeats a lot of the key to string mapping from the header bar.  We could move this to a `$scope.folderStatusTitle(folder)` function as well and reuse that in both HTML places.

Reply:
Done. I'm now going to work on the HTML part. It's a bit of a mess, so I need to make sure not to break anything 😅.

Reply:
All moved to the controller now. Please have a look when you find some time 🙂.